extensions/oidc/deployment/src/test/resources/application-dev-mode.properties

```diff
@@ -9,7 +9,7 @@ quarkus.oidc.application-type=web-app
 quarkus.oidc.logout.path=/protected/logout
 quarkus.oidc.authentication.pkce-required=true
 quarkus.log.category."org.htmlunit".level=ERROR
-quarkus.log.category."io.quarkus.oidc.runtime.TenantConfigContext".level=DEBUG
+quarkus.log.category."io.quarkus.oidc.runtime.TenantConfigContextImpl".level=DEBUG
```

Comment:
This is totally unrelated, but I wonder, if it would be nice to  point to the interface and have all interface implementations logging ?

Reply:
np, I'll do that. To be frank, I don't know what is customary

Reply:
let's wait if you require additional changes, if not, I suggest we can simply do it together with encapsulation of dynamic tenants, it will be related work that can be done quickly, especially as @snazy did most of the work already